src/bootstrap.js

```diff
@@ -105,7 +105,8 @@ export const defaultConfig = {
 	js: true,
 	css: true,
 	pushstate: false,
-	timeout: undefined
+	timeout: undefined,
+	dntSupport: true
```

Comment:
I think you want to set a default here too:
https://github.com/evolv-ai/asset-manager/blob/c1609f0e5d4e4b79256a4b5c3af5e9803cfcad1a/src/build-config.js#L49

```typescript
	const config = {
		dntSupport: true
	};
```